api/envoy/extensions/transport_sockets/starttls/v3/starttls.proto

```diff
@@ -29,7 +29,20 @@ message StartTlsConfig {
   // (optional) Configuration for clear-text socket used at the beginning of the session.
   raw_buffer.v3.RawBuffer cleartext_socket_config = 1;
 
-  // Configuration for TLS socket.
+  // Configuration for a downstream TLS socket.
   transport_sockets.tls.v3.DownstreamTlsContext tls_socket_config = 2
       [(validate.rules).message = {required: true}];
 }
+
+// Configuration for StartTls transport socket.
+// StartTls transport socket wraps two sockets:
+// - raw_buffer socket which is used at the beginning of the session
```

Comment:
Dashes `-` don't render as bullet points in the docs. See: https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/starttls/v3/starttls.proto.html
Use `*` instead.